src/core/Akka.Remote.TestKit/packages.config

```diff
@@ -1,9 +1,47 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="DotNetty.Buffers" version="0.4.3" targetFramework="net45" />
+  <package id="DotNetty.Codecs" version="0.4.3" targetFramework="net45" />
+  <package id="DotNetty.Common" version="0.4.3" targetFramework="net45" />
+  <package id="DotNetty.Transport" version="0.4.3" targetFramework="net45" />
   <package id="Google.ProtocolBuffers" version="2.4.1.555" targetFramework="net45" />
-  <package id="Helios" version="2.1.2" targetFramework="net45" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.1.0" targetFramework="net45" />
```

Comment:
Need to prune all of this crap out so it doesn't get pulled into the `.nuspec` metadata for `Akka.Remote.TestKit` as explicit dependencies.